Cogs.Design/CsSimpleType.md

```diff
@@ -0,0 +1,76 @@
+# C# Simple Type Mappings
+
+## boolean
+built-in bool type
+
+## string
+built-in string class with StringLength attribute
+
+## decimal
+built-in decimal type with Range and ExclusiveRange attributes
+
+## float
+built-in float type with Range and ExclusiveRange attributes
+
+## double
+built-in double type with Range and ExclusiveRange attributes
+
+## duration
+built-in timespan type
+
+## dateTime
+built-in DateTimeOffset type
+
+## time
+built-in DateTimeOffset type 
+
+## date
+built-in DateTimeOffset type wrapped in class inheriting from DateSuper
+
+## gYearMonth
+built-in tuple type containing two ints
+
+## gYear
+built-in int type with Range and ExclusiveRange attributes
+
+## gYearDay
+built-in tuple type containing two ints
+
+## gDay
+built-in int type with Range and ExclusiveRange attributes
+
+## gMonth
+built-in int type with Range and ExclusiveRange attributes
+
+## anyURI
+built-in Uri class with StringLength attribute
+
+## language
+built-in string class
+
+## integer
+built-in int type with Range and ExclusiveRange attributes
+
+## nonPositiveInteger
+built-in int type with Range and ExclusiveRange attributes
+
+## negativeInteger
+built-in int type with Range and ExclusiveRange attributes
+
+## long
+built-in long type with Range and ExclusiveRange attributes
+
+## int
+built-in int type with Range and ExclusiveRange attributes
+
+## nonNegativeInteger
+built-in uint type with Range and ExclusiveRange attributes
+
+## unsignedLong
+built-in ulong type with Range and ExclusiveRange attributes
+
+## positiveInteger
+built-in uint type with Range and ExclusiveRange attributes
+
+## cogsDate
```

Comment:
We can define this as a CogsDate type, which we can generate automatically along with the c# classes